wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -115,7 +113,7 @@
                 SUPPORTS_PLATFORM = true;
                 SYNC_FLAGS.add(MasternodeSync.SYNC_FLAGS.SYNC_HEADERS_MN_LIST_FIRST);
                 SYNC_FLAGS.add(MasternodeSync.SYNC_FLAGS.SYNC_BLOCKS_AFTER_PREPROCESSING);
-                FAUCET_URL = "http://faucet.krupnik.networks.dash.org/";
+                org.dash.wallet.common.Constants.FAUCET_URL = "http://faucet.krupnik.networks.dash.org/";
```

Comment:
This is a bit weird, but we need `FAUCET_URL` in another module.
I think the correct approach to consider for the future is using DI and having an interface with all necessary network-specific values, while the implementation will be provided depending on the build flavor.